prow/plugins/releasenote/releasenote.go

```diff
@@ -219,6 +219,11 @@ func handlePullRequest(pc plugins.Agent, pr github.PullRequestEvent) error {
 }
 
 func shouldHandlePR(pr *github.PullRequestEvent) bool {
+	//Do not handle the PR when it's merged
+	if pr.PullRequest.Merged {
```

Comment:
I think it's safe to say do not handle the PR since it's merged.  HandlePR function is all about adding and removing labels (also posting comments related to labels), so it's not needed for a merged PR.